cms/templates/index.html

```diff
@@ -552,6 +552,14 @@ <h3 class="title title-3">${_('Need help?')}</h3>
 
   %endif
 </div>
+
+${static.renderReact(
+  component="HelloWorld",
+  props={
+    'foo': 'bar'
+  }
+)}
+
```

Comment:
Example usage of the `renderReact` def. I'll pull this out and apply it to some real code (probs `StudioIndex` below) once I get a first pass thumb.